### ISSUE TITLE
Unable to run after installing on XUbuntu 18 (unknown attribute "days" in arrow.py)

### ISSUE THREAD

**Issue by AndreRisnes** [TailorDev/Watson#302]
Hi, I'm unable to run watson after installing it:

```
$ cat /etc/issue
Ubuntu 18.04.3 LTS
$ python3
Python 3.6.8 (default, Jan 14 2019, 11:02:34) 
^D
$ sudo pip3 install td-watson
...
$ watson start world-domination +cats
Traceback (most recent call last):
  File "/usr/local/bin/watson", line 7, in <module>
    from watson.__main__ import cli
  File "/usr/local/lib/python3.6/dist-packages/watson/__main__.py", line 2, in <module>
    from watson import cli
  File "/usr/local/lib/python3.6/dist-packages/watson/cli.py", line 364, in <module>
    k: get_start_time_for_period(k) for k in _SHORTCUT_OPTIONS
  File "/usr/local/lib/python3.6/dist-packages/watson/cli.py", line 364, in <dictcomp>
    k: get_start_time_for_period(k) for k in _SHORTCUT_OPTIONS
  File "/usr/local/lib/python3.6/dist-packages/watson/utils.py", line 148, in get_start_time_for_period
    start_time = arrow.Arrow.fromdate(now.replace(days=-weekday).date())
  File "/usr/local/lib/python3.6/dist-packages/arrow/arrow.py", line 590, in replace
    raise AttributeError('unknown attribute: "{}"'.format(key))
AttributeError: unknown attribute: "days"
```

This is the version of `arrow` that's installed:

``` $ pip3 show arrow
Name: arrow
Version: 0.14.5
```

**Comment by davidag** [comment]
This is fixed in #300, but a new version has not been released yet. Let's hope it will be soon :crossed_fingers: :relaxed: 

In the mean time, you can fix it by forcing a previous version of Arrow:

    $ sudo pip3 install arrow<=0.14.4

I hope this helps!

**Comment by AndreRisnes** [comment]
Thanks, that works!
